display_books_aysnc-await.js

```diff
@@ -0,0 +1,69 @@
+let BookData = new Array();
+fetchBookData();
+
+async function fetchBookData() {
+  try {
+    const response = await fetch("random_books.json");
+    BookData = await response.json();
+    insertDataInHtml(BookData);
+  } catch (error) {
+    console.log("Error in Fetching data", error);
+  }
+}
+
+document.getElementById("btn-search-id").onclick = IdButton;
+document.getElementById("btn-search-genre").onclick = genreButton;
+document.getElementById("btn-search-price").onclick = priceButton;
+
+function IdButton(event) {
+  event.preventDefault();
+  const idValue = document.getElementById("input-search-id").value;
+  if (idValue.trim().length <= 0) return;
+  const matchingBooks = BookData.filter(
+    (book) => book.bookId.toString() === idValue
+  );
+  insertDataInHtml(matchingBooks);
+}
+
+function genreButton(event) {
```

Comment:
use es6 syntax as much as possible at every instance

Reply:
Noted and applied wherever possible